src/lib/accessibility/Behaviors/Tree/GroupBehavior.ts

```diff
@@ -0,0 +1,20 @@
+import { IAccessibilityBehavior, ComponentState } from '../../interfaces'
+import { AbstractBehavior } from '../AbstractBehavior'
+
+export class GroupBehavior extends AbstractBehavior<{}, {}>
+  implements IAccessibilityBehavior<{}, {}> {
+  constructor() {
+    super('group')
+  }
+
+  private attributes = {
+    'ms-acc-behavior': this.name,
+    role: 'group',
+  }
+
+  public generateAriaAttributes(props, state): object {
```

Comment:
are we planning to introduce real types in the future? I get it that this might be just a prototype for now, wanted to know what's the roadmap here

Reply:
Right, this is a prototype. We still haven't decided on one solid approach. Anyway, types should be there, in the future.